src/OrbitQt/CallTreeWidget.h

```diff
@@ -133,6 +138,7 @@ class CallTreeWidget : public QWidget {
     kDone = 2            // The columns have been resized (once)
   };
   ColumnResizingState column_resizing_state_ = ColumnResizingState::kInitial;
+  bool resize_thread_or_function_column_on_row_expanded_ = true;
```

Comment:
The usage of this variable is a bit strange to me but I guess is the best way to solve this problem with signals.

Reply:
Yeah, I don't like it that much either, but something like this is somewhat necessary as the performance difference is measurable.
By the way, a not-so-different alternative would be to not use this field but call `disconnect` and `connect`. I went for the variable as in theory `OnRowExpanded` could do something else that is cheap and needs to be done at all times. But the general principle is the same.